packages/binding.core/spec/enableDisableBehaviors.ts

```diff
@@ -29,33 +30,38 @@ describe('Binding: Enable/Disable', function () {
     var myObservable = observable()
     testNode.innerHTML = "<input data-bind='enable:myModelProperty()' />"
     applyBindings({ myModelProperty: myObservable }, testNode)
-
-    expect(testNode.childNodes[0].disabled).toEqual(true)
+    var input = testNode.children[0] as HTMLInputElement
```

Comment:
should be `const`.  small thing, but just to prevent hoisting since it's used a bunch in the other functions here where it could impact outcomes

Reply:
Done in https://github.com/Auge19/tko/pull/4